lib/logstash/inputs/redis.rb

```diff
@@ -29,6 +29,9 @@ module LogStash module Inputs class Redis < LogStash::Inputs::Threadable
   # The port to connect on.
   config :port, :validate => :number, :default => 6379
 
+  # SSL
+  config :ssl, :validate => :boolean, :default => false
```

Comment:
this must be documented in the `docs/index.asciidoc` file, as that is the source of truth for the plugin's documentation

Reply:
Added the requested documentation

Reply:
@jsvd Can we merge it?